.github/workflows/delivery/chocolatey/tools/LICENSE.txt

```diff
@@ -0,0 +1,206 @@
+
+From: https://github.com/buildpacks/pack/blob/main/LICENSE
```

Comment:
Could we generated this file by copying it over from the repo instead of having two copies?